testsuite/lees_edwards.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import espressomd as md
+import unittest as ut
+
+@ut.skipIf(not md.has_features(['LEES_EDWARDS']),
+'Features not available, skipping test!')
+
+class Lees_Edwards_test(ut.TestCase):
+
+  def test(self):
+    
```

Comment:
Please provide a docstring for the test method that describes what is beeing tested and how.

Reply:
Maybe devide into several tests for clarification.